deployment/terraform/assets/cluster.tf

```diff
@@ -40,6 +40,8 @@ locals {
 data "aws_subnets" "selected" {
   filter {
     name   = "vpc-id"
+    // TODO: Any specific reason we use a specific VPC for ES only here?
+    // Should we use the `cluster_vpc_id` variable instead and remove `es_vpc`?
```

Comment:
I remember VPC being a key detail when deploying ES, but that has likely preceded our ability to set one so we may be able to use the same variable. That said, the best thing would be to deploy the changes and see that everything works.